localstack/config.py

```diff
@@ -762,6 +762,11 @@ def populate_edge_configuration(
 # get-function call.
 INTERNAL_RESOURCE_ACCOUNT = os.environ.get("INTERNAL_RESOURCE_ACCOUNT") or "949334387222"
 
+# Determine which implementation to use for the event rule / event filtering engine used by multiple services:
+# EventBridge, EventBridge Pipes, Lambda Event Source Mapping, SNS
+# Options: provider (default) | java
+EVENT_RULE_ENGINE = os.environ.get("EVENT_RULE_ENGINE", "").strip()
```

Comment:
✅ CI run with JPype enabled using `java` here in [017bae5](https://github.com/localstack/localstack/pull/10615/commits/017bae519040155366f4e43feecddd790eb19907)